src/app/builders/components/Table/Cell/BuilderNameCell/BuilderNameCell.tsx

```diff
@@ -99,18 +100,21 @@ export const BuilderNameCell: FC<BuilderNameCellProps> = ({
   className,
 }) => {
   const builderState = getBuilderStateForIcon(builder)
```

Comment:
In this method we are also missing the `selfPaused` icon

Reply:
we don't have self-paused icon in the designs. only the shield and the hourglass